.gitignore

```diff
@@ -33,6 +34,7 @@ nosetests.xml
 .mr.developer.cfg
 .project
 .pydevproject
+.idea
```

Comment:
This is fine, but things like this should really go in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).